homeassistant/components/litejet/switch.py

```diff
@@ -56,6 +61,17 @@ async def async_will_remove_from_hass(self) -> None:
         self._lj.unsubscribe(self._on_switch_released)
         self._lj.unsubscribe(self._on_connected_changed)
 
+    @property
+    def device_info(self) -> DeviceInfo:
+        """Return the device info."""
+        # Keypad #1 has switches 1-6, #2 has 7-12, ...
+        keypad_number = int((self._index - 1) / 6) + 1
+        return DeviceInfo(
+            identifiers={(DOMAIN, f"{self._entry_id}_keypad_{keypad_number}")},
+            name=f"Keypad #{keypad_number}",
```

Comment:
This conflicts with what's set in in `light.py` and `scene.py`.
Or does a centralite device which generates switch entities not have any light or sceneentities?

Reply:
Right. Think of LiteJet as a fixed-in-stone hub system that has no extensibility. It has a central panel that every keypad and light wire run back to. The keypads and lights can be in different rooms (HA areas) so I made them their own devices. Scenes can act on multiple of these lights so I attached them to the hub device.